### ISSUE TITLE
Change test cases to json

### ISSUE THREAD

**Issue by mrp089** [SimVascular/svZeroDSolver-Archived#37]
Depends on #14. Change all test cases to the new `json` format. Retain all comments in the file. @JonathanPham Can you use your pipeline to convert these automatically?

**Comment by mrp089** [comment]
It looks like the comments of the original test cases got lost in the conversion. They were quite useful in describing what a test case does and contained some analytical derivations of the results. 

@JonathanPham, can you add those back in? Only the useful ones, not the documentation of the input file. Thank you!!

**Comment by JonathanPham** [comment]
Added back the description comments for the test cases in pull request #42.

**Comment by mrp089** [comment]
I think I got mixed up between `0D` and `1D`. It looks like our `0D` test cases never had a detailed explanation. I think it would make sense to add documentation to all test cases on how we calculated the analytical reference solution (similar to https://github.com/SimVascular/svOneDSolver/tree/master/test). It doesn't need to be super extensive. In most cases a one-line formula for, e.g., the outlet pressure would be enough. I think that would help new users understand what we're computing.
